gui/app/controls/readings/PipDisplay.qml

```diff
@@ -7,5 +7,7 @@ ParameterDisplay {
     parameterName: qsTr("PIP")
     parameterUnit: qsTr("cmH<sub>2</sub>O")
     parameterValue: GuiStateContainer.measured_pip.toString()
-    alarmPriority: GuiStateContainer.alarmManager.pipExceededAlarm.effectiveVisualPriority
+    alarmPriority: Math.max(
```

Comment:
Not sure yet whether it makes sense to deduplicate this with the similar expression in PressureGraph.qml. I'd wait with that until we have any alarms about PEEP too.